src/contexts/InputsContext.js

```diff
@@ -204,6 +211,24 @@ function reduce(state, action) {
         draft.currencySelectionModal.isVisible = !state.currencySelectionModal.isVisible;
       });
 
+    case 'TOGGLE_ADD_LIQUIDITY_MODAL':
```

Comment:
it's better to use constants for string values which we use in more places. Could you declare const values and import them in other modules?